src/DetailsView/Styles/detailsview.scss

```diff
@@ -888,16 +889,13 @@ div.insights-file-issue-details-dialog-container {
                 }
                 .issues-table {
                     height: 100%;
-                    width: 100%;
+                    width: calc(100% - 48px);
                     display: flex;
                     flex-direction: column;
                     margin-top: 24px;
-                    > h1,
-                    .issues-table-subtitle,
-                    .details-view-toggle,
-                    .details-disabled-message {
-                        margin-left: 24px;
-                    }
+
```

Comment:
Any specific reason, not to use single `margin` property like `margin: 24px 24px auto 24px` rather than use separate properties? Just curious.

Reply:
Good catch; will change.

Reply:
Personally, I think the split versions of compound properties like this are more readable; if they're split, I think it's much harder to overlook two of them being transposed and much less likely that a future will need to look up documentation to understand the setting